docs/user/aws/iam_after.md

```diff
@@ -0,0 +1,14 @@
+# IAM User: Revisited
+
+Once OpenShift is installed, the installer account no longer is required to have AdministratorAccess. This user account 
```

Comment:
"no longer is required to have" -> "no longer requires"?

Reply:
Fixed. Thanks!